api/swagger-ui/index.html

```diff
@@ -49,12 +49,16 @@
         plugins: [
           SwaggerUIBundle.plugins.DownloadUrl
         ],
-        layout: "StandaloneLayout"
+        layout: "StandaloneLayout",
+        urls: [{
+          url: "/swagger-ui/swagger.yaml",
+          name: "api-service"
+        }, {url: "/configuration-service/swagger-ui/swagger.yaml", name: "configuration-service"}],
```

Comment:
Not a major issue, but let's make this look cleaner by adding some newlines:
```javascript
        urls: [{
          url: "/swagger-ui/swagger.yaml",
          name: "api-service"
        }, {
          url: "/configuration-service/swagger-ui/swagger.yaml", 
          name: "configuration-service"
       }],
```

Reply:
Done.